packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -44,6 +44,12 @@ This is what happens under the hood:
    `websiteBucket`). If there is more than one source, the sources will be
    downloaded and merged pre-deployment at this step.
 
+**IMPORTANT** The `aws-s3-deployment` module is only intended to be used with
+assets built by the CDK CLI and Zip files from sources you trust. If you use
+`Source.bucket()` to reference a zip file from an untrusted source, it might
+execute arbitrary code in the context of the Lambda Function and read or write
```

Comment:
I think slightly more information about what "context" means here. Can the code use the function's assumed permissions to access AWS resources? Or just mess with the runtime of the function? Or...

Reply:
Both. I had hoped the word "context" would be clear enough, but I guess not :)